src/hooks/useTheme.test.tsx

```diff
@@ -14,18 +16,18 @@ const renderHookInContext = async (theme?: Theme) => {
 };
 
 test('returns theme', async () => {
-  const { result } = await renderHookInContext();
+  const {
+    result: { current: theme },
+  } = await renderHookInContext();
 
-  expect(result.current.theme).toBeInstanceOf(Theme);
-  expect(result.current.theme).toEqual(baseTheme);
+  expect(theme.colors.onBackground).toBe(baseTheme.colors.onBackground);
+  expect(theme.colors.background).toBe(baseTheme.colors.background);
```

Comment:
Paper theme has other defaults that our baseTheme doesn't worry about.  So I'm just testing a couple properties rather than asserting that the whole thing is the same.